src/Command/RoutesCommand.php

```diff
@@ -40,9 +40,10 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
             $header[] = 'Defaults';
         }
 
-        $output = [];
+        $routeCollection = Router::routes();
```

Comment:
This name is a bit misleading since we have a `RouteCollection` class while this is an array of routes :)

Reply:
changed it now to `$availableRoutes`